code/modules/power/apc/apc_main.dm

```diff
@@ -663,26 +671,47 @@
 		update_appearance(UPDATE_ICON)
 		update()
 
+/**
+ * Used for cell_5k apc helper, which installs 5k cell into apc.
+ */
```

Comment:
these can just be three slashes like

```
/// Used for cell_5k apc helper, which installs 5k cell into apc.
```

no need to take up more than one line for a simple message such as this.